### PR TITLE
fix cabforum's documents URL

### LINKED ISSUE #414: URLs
_Comment by @larseggert_

These URLs in the document did not return content:

* https://www.cabforum.org/documents.html

### PR DESCRIPTION
**Pull request by thomas-fossati**
Fix #414

Signed-off-by: Thomas Fossati <thomas.fossati@arm.com>

### CODE REVIEW

**Comment by stpeter** [comment]
Although this looks fine, we might want to ask the RFC Editor team about the stability of this reference (e.g., should we point to https://www.cabforum.org/ alone and let people navigate from there?).

**Comment by stpeter** [comment]
Yes, this is another thing to talk about with the RFC Editor team. We might want to reference "latest" rather than a specific version (which, after all, is merely a snapshot).

**Review by yaronf — CHANGES_REQUESTED** [review]
Suggest to keep open while we have this discussion.

File: BCP195bis/draft-ietf-uta-rfc7525bis.md [hunk 1]
```diff
@@ -809,7 +809,9 @@ Registry"), within the
...
 revision of {{NIST.SP.800-56A}}. 
 
-When using RSA, servers MUST authenticate using certificates with at least a 2048-bit modulus for the public key.  In addition, the use of the SHA-256 hash algorithm is RECOMMENDED and SHA-1 or MD5 MUST NOT be used ({{!RFC9155}}, and see {{CAB-Baseline}} for more details). Clients MUST indicate to servers that they request SHA-256, by using the "Signature Algorithms" extension defined in TLS 1.2. For TLS 1.3, the same requirement is already specified by {{RFC8446}}.
+When using RSA, servers MUST authenticate using certificates with at least a 2048-bit modulus for the public key.  In addition, the use of the SHA-256 hash algorithm is RECOMMENDED and SHA-1 or MD5 MUST NOT be used ({{!RFC9155}}, and see {{CAB-Baseline}}[^live-ref-question] for more details). Clients MUST indicate to servers that they request SHA-256, by using the "Signature Algorithms" extension defined in TLS 1.2. For TLS 1.3, the same requirement is already specified by {{RFC8446}}.
+
+[^live-ref-question]: RFC Editor: We are looking for advice on how to best handle this constantly updated guidance.  In particular: which URL to use, which (if any) version to reference
```
Comment (stpeter) [inline BCP195bis/draft-ietf-uta-rfc7525bis.md:814]:
```suggestion
Note to RFC Editor: We are looking for advice on how to best handle this constantly updated guidance from the CA / Browser Forum.  In particular: which URL to use, which (if any) version to reference
```

**Comment by thomas-fossati** [comment]
> Suggest to keep open while we have this discussion.

@yaronf I think we can close this now (there's a note in-band for the RFC-Ed)

File: BCP195bis/draft-ietf-uta-rfc7525bis.md [hunk 2]
```diff
@@ -809,7 +809,9 @@ Registry"), within the
...
 
-When using RSA, servers MUST authenticate using certificates with at least a 2048-bit modulus for the public key.  In addition, the use of the SHA-256 hash algorithm is RECOMMENDED and SHA-1 or MD5 MUST NOT be used ({{!RFC9155}}, and see {{CAB-Baseline}} for more details). Clients MUST indicate to servers that they request SHA-256, by using the "Signature Algorithms" extension defined in TLS 1.2. For TLS 1.3, the same requirement is already specified by {{RFC8446}}.
+When using RSA, servers MUST authenticate using certificates with at least a 2048-bit modulus for the public key.  In addition, the use of the SHA-256 hash algorithm is RECOMMENDED and SHA-1 or MD5 MUST NOT be used ({{!RFC9155}}, and see {{CAB-Baseline}}[^live-ref-question] for more details). Clients MUST indicate to servers that they request SHA-256, by using the "Signature Algorithms" extension defined in TLS 1.2. For TLS 1.3, the same requirement is already specified by {{RFC8446}}.
+
+Note to RFC Editor: We are looking for advice on how to best handle this constantly updated guidance from the CA / Browser Forum.  In particular: which URL to use, which (if any) version to reference
```
Comment (yaronf) [inline BCP195bis/draft-ietf-uta-rfc7525bis.md:814]:
```suggestion
Note to RFC Editor: We are looking for advice on how to best handle this constantly updated guidance from the CA/Browser Forum.  In particular: which URL to use, which (if any) version to reference
```

File: BCP195bis/draft-ietf-uta-rfc7525bis.md [hunk 3]
```diff
@@ -809,7 +809,9 @@ Registry"), within the
    groups.  Curves of less than 224 bits MUST NOT be used. This recommendation is in-line with the latest
 revision of {{NIST.SP.800-56A}}. 
 
-When using RSA, servers MUST authenticate using certificates with at least a 2048-bit modulus for the public key.  In addition, the use of the SHA-256 hash algorithm is RECOMMENDED and SHA-1 or MD5 MUST NOT be used ({{!RFC9155}}, and see {{CAB-Baseline}} for more details). Clients MUST indicate to servers that they request SHA-256, by using the "Signature Algorithms" extension defined in TLS 1.2. For TLS 1.3, the same requirement is already specified by {{RFC8446}}.
+When using RSA, servers MUST authenticate using certificates with at least a 2048-bit modulus for the public key.  In addition, the use of the SHA-256 hash algorithm is RECOMMENDED and SHA-1 or MD5 MUST NOT be used ({{!RFC9155}}, and see {{CAB-Baseline}}[^live-ref-question] for more details). Clients MUST indicate to servers that they request SHA-256, by using the "Signature Algorithms" extension defined in TLS 1.2. For TLS 1.3, the same requirement is already specified by {{RFC8446}}.
```
Comment (yaronf) [inline BCP195bis/draft-ietf-uta-rfc7525bis.md:812]: The "live-ref-question" thing doesn't render.

**Review by yaronf — CHANGES_REQUESTED** [review]

File: BCP195bis/draft-ietf-uta-rfc7525bis.md (hunk 3 above)
Comment (thomas-fossati) [reply BCP195bis/draft-ietf-uta-rfc7525bis.md:812]: oh right I didn't notice that [2fb0e66](https://github.com/yaronf/I-D/pull/428/commits/2fb0e662ef223b8201d7914aa2be7850b3b260a0) had removed the anchor.

will fix it (along with the other comment you have below)

Comment (thomas-fossati) [reply BCP195bis/draft-ietf-uta-rfc7525bis.md:812]: done in [f229a1e](https://github.com/yaronf/I-D/pull/428/commits/f229a1ee1ee411caf62e54526778bd1a2d1edac0)

File: BCP195bis/draft-ietf-uta-rfc7525bis.md (hunk 2 above)
Comment (thomas-fossati) [reply BCP195bis/draft-ietf-uta-rfc7525bis.md]: also done in [f229a1e](https://github.com/yaronf/I-D/pull/428/commits/f229a1ee1ee411caf62e54526778bd1a2d1edac0)

**Review by yaronf — CHANGES_REQUESTED** [review]
@thomas-fossati Sorry, the Markdown construct you're using comes out really badly in HTML and even worse in TXT. I suggest to revert to simple text for the comment.

**Comment by thomas-fossati** [comment]
> @thomas-fossati Sorry, the Markdown construct you're using comes out really badly in HTML and even worse in TXT. I suggest to revert to simple text for the comment.

[50a800c](https://github.com/yaronf/I-D/pull/428/commits/50a800ca66fdc4f79013e4ffd426fea186b5babd) should render OK

**Review by yaronf — APPROVED** [review]

**Pull request merged**